webapp/src/js/services/enketo.js

```diff
@@ -341,6 +341,7 @@ angular.module('inboxServices').service('Enketo',
           .then(function(form) {
             registerEditedListener(selector, editedListener);
             registerValuechangeListener(selector, valuechangeListener);
+            $window.debugFormModel = () => form.getModel().getStr();
```

Comment:
I get a deprecation message when I run this:
```
form.getModel() is deprecated. Use form.model instead.
```

I changed it to `form.model.getStr()` and it works just the same, bar the deprecation message.